spec/features/users/registration_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe "User can Register" do
+  it "shows a form to enter information and once logged in redirects to profile page" do
+
+    visit "/merchants"
+    within 'nav' do
+      click_on "Register"
+    end
+    expect(current_path).to eq("/register")
+
+    page.has_field?(:name)
+    page.has_field?(:city)
+    page.has_field?(:state)
+    page.has_field?(:zip)
+    page.has_field?(:email)
+    page.has_field?(:password)
+    page.has_field?(:confirm_password)
```

Comment:
This could be removed possibly for redundancy since we are filling in the same fields below.